src/common/reducers/user.ts

```diff
@@ -0,0 +1,32 @@
+import {handleActions} from 'redux-actions';
+
+import {
+	LOGIN,
+	LOGOUT,
+	SET_USER_ACCESS_TOKEN,
+	SET_USER_REFRESH_TOKEN,
+} from '../constants/user';
+
+const initialState = {
+	isAuthenticated: false,
+	name: '',
+};
+
+export default handleActions({
+	[LOGIN]: (state: any) => ({
+		...state,
+		isAuthenticated: true,
+	}),
+	[LOGOUT]: (state: any) => ({
+		...state,
+		isAuthenticated: false,
+	}),
+	[SET_USER_ACCESS_TOKEN]: (state: any, {payload}) => ({
```

Comment:
if you use destructuring here, more logical way to took a token from payload, {payload: {token}}